src/Microsoft.DotNet.Interactive.SqlServer/Microsoft.DotNet.Interactive.SqlServer.csproj

```diff
@@ -20,6 +20,7 @@
  </ItemGroup>
 
  <ItemGroup>
+   <PackageReference Include="Microsoft.Azure.Kusto.Data" Version="9.3.1" />
```

Comment:
Added Kusto.Data to handle getting an AAD token for Kusto. In the future this can be refactored to use ADAL

Reply:
Is this needed any longer for SQL?